[ASM] - Expander - Expander v2 Pack enhancements - batch 1 (#27006)
demisto/content: Packs/CortexXpanse/ReleaseNotes/1_0_4.md

> 

Reviewer: Please make sure the release notes (especially prefixes) match the template: https://xsoar.pan.dev/docs/documentation/release-notes#general

#### Dashboards

##### New: Cortex Xpanse Overview Dashboard

> - This dashboard provides a high level view of incidents from Cortex Xpanse.

Reviewer: ```suggestion
- Added a dashboard that provides a high level view of incidents from Cortex Xpanse.
```

#### Incident Fields

- New: **Xpanse Protocol**

> - New: **Xpanse Service Validation**

Reviewer: ```suggestion
- Added the incident field **Xpanse Service Validation** which keeps track of service validation details.
```

- New: **Xpanse Service ID**

> - New: **Xpanse Provider**

Reviewer: ```suggestion
- Added the incident field **Xpanse Provider** which keeps track of the provider of the service.
```

> - New: **Xpanse Service Classifications**

Reviewer: ```suggestion
- Added the incident field **Xpanse Service Classifications** which keeps track of active service classifications.
```

#### Incident Types

- Added a new default playbook was added to **Xpanse Alert**.

#### Playbooks

##### New: Xpanse - Alert Enrichment

- Added a new playbook with additional enrichment via cloud service providers and other applicable IT and security tools.

##### New: Xpanse - Alert Handler

- Added a default Alert Handling playbook for Xpanse Alerts.

##### New: Xpanse - NMap - Detect Service

- Added a new playbook that looks at what ASM sub-type the alert is and uses NMap to do a validation scan.

##### New: Xpanse - Alert Self-Enrichment

- Added a new enrichment playbook on the alert itself using Cortex Xpanse APIs.

#### Widgets

##### New: Xpanse Incidents by Provider

- Added a new widget that includes provider names for all Xpanse Incidents.

##### New: Xpanse Problematic IPs

- Added a new widget that highlights the most common IPs involved in Xpanse Incidents.

##### New: Xpanse Incident Count by Business Unit

- Added a new widget that highlights the most common Business Units involved in Xpanse Incidents.

##### New: Unassigned Active Xpanse Incidents

> - Added a new widget that includes the number of Active Xpanse Incidents without an assignee.

Reviewer 2: ```suggestion
Added a new widget that includes the number of active Cortex Xpanse incidents without an assignee.
```

##### New: Assigned Active Xpanse Incidents

> - Added a new widget that includes the number of Active Xpanse Incidents with an assignee.

Reviewer 2: ```suggestion
Added a new widget that includes the number of active Cortex Xpanse incidents with an assignee.
```

##### New: Exposed Ports in Xpanse Incidents

> - Added a new widget that highlights the most common Xpanse Incidents by port.

Reviewer 2: ```suggestion
Added a new widget that highlights the most common Cortex Xpanse incidents by port.
```

##### New: Xpanse Incidents by Protocol

> - Added a new widget that highlights the most common Xpanse Incidents by protocol.

Reviewer 2: ```suggestion
Added a new widget that highlights the most common Cortex Xpanse incidents by protocol.
```

##### New: New Xpanse High & Critical Incidents

> - Added a new widget that includes a timeline view of new high and critical severity Xpanse Incidents.

Reviewer 2: ```suggestion
Added a new widget that includes a timeline view of new high and critical severity Cortex Xpanse incidents.
```

#### Layouts

##### Xpanse Alert Layout

> - Updated the Xpanse alert layout to include several new fields.

Reviewer 2: ```suggestion
Updated the Cortex Xpanse alert layout to include several new fields.
```
